conf/modules.config

```diff
@@ -283,23 +283,14 @@ process {
         ]
     }
 
-    withName: 'MAG_DEPTHS_PLOT|MAG_DEPTHS_SUMMARY|MAG_DEPTHS_PLOT_REFINED' {
+    withName: 'MAG_DEPTHS_PLOT|MAG_DEPTHS_SUMMARY' {
         publishDir = [
             path: { "${params.outdir}/GenomeBinning/depths/bins" },
             mode: params.publish_dir_mode,
             pattern: "*.{png,tsv}"
         ]
     }
 
-    withName: 'MAG_DEPTHS_SUMMARY_REFINED' {
-        ext.prefix = "bin_refined_depths_summary"
-        publishDir = [
-            path: { "${params.outdir}/GenomeBinning/depths/bins" },
-            mode: params.publish_dir_mode,
-            pattern: "*.{tsv}"
-        ]
-    }
-
```

Comment:
Reason why both have deleted (make sense to collapse into one `withName` though, if that was the original purpose)?

Reply:
I moved all the MAG_DEPTHS processes to a new subworkflow that runs after bin refinement, and operates depending on which bins are kept as defined in `params.postbinning_input`, so there's no separate calculation of refined bin depths any more (maybe there should be?). Otherwise, yes, just collapsing into one withName!

Reply:
Ah yes good point! Nope, logic makes sense :+1: